proto/ecosystem-incentive/ecosystem_incentive.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+package ununifi.ecosystemincentive;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+
+option go_package = "github.com/UnUniFi/chain/x/ecosystem-incentive/types";
+
+message IncentiveUnit {
+  string id = 1 [
+    (gogoproto.moretags) = "yaml:\"id\""
+  ];
+  repeated SubjectInfo subject_info_list = 2 [
```

Comment:
How many subject info list could be available per incentive unit? I think there should be limitation here.

Reply:
hmm, i didn't think about it since there's no reason to bind it only from the perspective of the function.
but, yeah, i think you're right.
what is the number that you think the maximum group of people build frontend service in general?

Reply:
I implemented the logic using the default value for the max number of subject info in a unit is 15.